parts/openshift/unstable/openshiftmasterscript.sh

```diff
@@ -53,13 +111,10 @@ oc adm create-bootstrap-policy-file --filename=/etc/origin/master/policy.json
 
 cp /etc/origin/node/ca.crt /etc/pki/ca-trust/source/anchors/openshift-ca.crt
 update-ca-trust
+}
 
-# FIXME: It is horrible that we're installing az.  Try to avoid adding
-# additional functionality in this script that requires it.  One route to remove
-# this code is to bake this script into the base image, then pass in parameters
-# such as the registry storage account name and key direct from ARM.
-rpm -i https://packages.microsoft.com/yumrepos/azure-cli/azure-cli-2.0.31-1.el7.x86_64.rpm
-
+# configure azure cloud provider
+function configure_cloud_provider() {
 set +x
 . <(sed -e 's/: */=/' /etc/origin/cloudprovider/azure.conf)
 az login --service-principal -u "$aadClientId" -p "$aadClientSecret" --tenant "$aadTenantId" &>/dev/null
```

Comment:
this one need better error handling as if variables are passed wrong it errors to /dev/null